src/assets/js/features.js

```diff
@@ -0,0 +1,15 @@
+import Fetcher from '@/assets/js/fetcher';
+export default class Features {
+  constructor() {
+    this.fetcher = new Fetcher({ failoverOn: [302] });
+  }
+  async get() {
+    try {
+      const result = await this.fetcher.fetch('/config/features.json');
+      return await result.json();
+    } catch (e) {
+      console.error(e.message);
```

Comment:
This should not fail, but we could add default proxy object which returns false for all features and fall back to that.